qa/tasks/cephadm.py

```diff
@@ -100,6 +100,32 @@ def update_archive_setting(ctx, key, value):
         yaml.safe_dump(info_yaml, info_file, default_flow_style=False)
 
 
+@contextlib.contextmanager
+def nvmeof_context(ctx, config):
+    (remote,) = ctx.cluster.only('host.a').remotes.keys()
+    ip_address = remote.ip_address
+    gateway_addr = remote.sh("systemctl list-units | grep nvmeof.mypool | awk '{print $1}' | xargs echo | cut -d '@' -f2- | sed 's/.\{8\}$//'",
```

Comment:
It would be nice to avoid hard coding the pool name here and roles (`host.a`, `client.1`) above and below.